src/commands/fishy/fishy.ts

```diff
@@ -96,22 +97,22 @@ export default class FishyCommand extends Command {
       if (isDmChannel(message) || isInChannel(message, allChannels.fishy) || !isHomeGuild(message)) {
         if (canFish === true) {
           const total = fish.reduce((acc, cur) => acc + cur.weight, 0);
-          const threshold = Math.random() * total;
+          const threshold = crypto.randomInt(0, total);
 
           let sum = 0;
           const group = fish.find((group) => {
             sum += group.weight;
             return sum >= threshold;
           });
 
-          const index = Math.floor(Math.random() * group.puns.length);
+          const index = crypto.randomInt(0, group.puns.length - 1);
           const fishPun = group.puns[index];
 
           if (group.type === 'totalTrash') {
             var amount = 0;
             var reply = group.catch;
           } else {
-            var amount = Math.floor(Math.random() * (group.max - group.min) + group.min);
+            var amount = crypto.randomInt(group.min, group.max);
```

Comment:
*max* is non-inclusive, so you should add one to get the full range of fishy values